src/routes/(app)/product/[id]/+page.server.ts

```diff
@@ -107,6 +108,24 @@ export const load = async ({ params, locals }) => {
 			sort: { percentage: -1 }
 		}
 	);
+	const jsonLd = {
+		'@context': 'https://schema.org/',
+		'@type': 'Product',
+		name: product.name,
+		image: `${ORIGIN}/picture/raw/${pictures[0]._id}/format/${pictures[0].storage.formats[0].width}`,
+		description: product.description,
+
+		brand: {
+			'@type': 'Brand',
+			name: product.name
+		},
```

Comment:
No, we should only specfiy `brand` if we have the brand of the product (and it's not information we store at the moment)